config/routes.rb

```diff
@@ -16,6 +16,12 @@
     root to: "auth/sessions#new"
   end
 
+  get '/installer/options', to: 'installer#options'
+  get '/installer/master', to: 'installer#master'
+  get '/nodes/add', to: 'nodes#add'
+  get '/nodes', to: 'nodes#list'
+  get '/config', to: 'config#index'
```

Comment:
`/installer/config` is intended to be the first page the user will see when Yast installation is over.
`/installer/master` is the following page in the flow, where the master is selected by the user. Once clicked on Next, the installation process with proceed and once completed, move to the next page.
`/nodes` is the list of available up-and-running nodes in the cluster, and the page the user would see once logged into the system, once it's up and running (honestly, I think this should be the real dashboard, with the tachometers and other control components... but is up to the product owner and designer)
`/nodes/add` is the page where new nodes will be displayed, selected and added to the existing cluster.
`/config` is the configuration page for velum where, among other things, the autoupdate checkbox should be displayed. Options from `/installer/config` could be displayed and modified (Federica?).